engines/access/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
-# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
-add_engine access "Access" yes
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
+add_engine access "Access" yes "" "" "" "mt32emu"
```

Comment:
I wonder if there should be a more general "midi" component that covers FluidSynth and EAS as well.

Reply:
Right. It could make sense. I will think about it. The thing is that it will require a bespoke code for enabling three features if this component is enabled, ano moreover, they should turn into `auto`, not `yes`, so the detection code is launched against each of them.